platform/node/index.js

```diff
@@ -19,7 +19,12 @@ var Map = function(options) {
     return new constructor(Object.assign(options, {
         request: function(req) {
             request(req, function() {
-                req.respond.apply(req, arguments);
+                var args = arguments;
+                // Protect ourselves from `request` implementations that try to release Zalgo.
```

Comment:
Yeah, I saw the link, but even then it took me a while to understand that it refers to a meme. Thanks for updating!